### DISCUSSION TITLE
Smooth Starting with Parallaxx (Re-Focus)

### DISCUSSION THREAD

**Discussion by akanesh** [rocksdanister/lively#1974] [Ideas]
Hi,

maybe that's an easy idea to implement:

When focus comes again to a parallax-theme I see a kind of stuttering when the animation starts (i7 / 16GB) / SSD). I would like to have a soft animation-restart (from 1% to 100% speed) maybe after a period (1 to ? seconds), every time the focus comes back to the parallax-theme. 

That would make the whole, great looking thing, even smoother.

Does anybody have the same issues and likes this idea?

Greatings ...

**Comment by rocksdanister** [comment]
If anyone plan to implement this look into wallpaper pause event:
https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data#--pause-event
